paddle/fluid/framework/ngraph_operator.cc

```diff
@@ -302,21 +302,24 @@ void NgraphEngine::GetNgInputShape(std::shared_ptr<OperatorBase> op) {
 }
 
 void NgraphEngine::BuildNgNodes() {
-  for (auto& var_name : var_out_) {
-    if (var_node_map_->find(var_name) == var_node_map_->end()) {
-      auto* var = scope_.FindVar(var_name);
-      if (var && var->IsType<LoDTensor>()) {
-        auto* tensor_pd = GetLoDTensorOrSelectedRowsValueFromVar(*var);
-        auto& ddim = tensor_pd->dims();
-        auto ng_shape = Ddim2Shape(ddim);
-        auto ng_type = var_type_map_.at(var_name);
-        auto prm =
-            std::make_shared<ngraph::op::Parameter>(ng_type, ng_shape, true);
-        (*var_node_map_)[var_name] = prm;
+  for (auto& op : fused_ops_) {
```

Comment:
why do you update here for adding conv2d_op?

Reply:
The following list had not being completed as of yet - basically is ancient as the hill. To alleviate this utterance, I want to say that the list of the output tensors is completed right now. Notably, We're having these changes added to other pull-request.

Reply:
The following list had not been completed as of yet - basically it is as ancient as the hill. To improve it, I want to say that the list of the output tensors is completed right now. Thus, we will be able to get some information about the output tensors, i.e the size of the tensors. Also, we are going to have these changes added to other pull-request.

Reply:
@mozga-intel This change is no longer needed as PR #14948 merged. The ngraph_operator.cc is pending to be removed per PR #15537. Also can you address the conflict from PR #15538?

Reply:
@baojun-nervana Great!, I just saw, I will take these changes into account to resolve these conflicts.